### ISSUE TITLE
Error loading config.json. Please check your config.json file: ReferenceError: Headers is not defined

### ISSUE THREAD

**Issue by slice-sarthakd** [continuedev/continue#875]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: MacOS
- Continue: pre-released version and latest version
- IDE: VsCode
```

### Description

While adding a localmodel of ollama it throws error
<img width="1257" alt="Screenshot 2024-02-18 at 5 19 17 PM" src="https://github.com/continuedev/continue/assets/106574498/d89af4b2-c44d-4686-9127-745806fff082">

```
{
  "models": [
    {
      "title": "GPT-4 (Free Trial)",
      "provider": "free-trial",
      "model": "gpt-4"
    },
    {
      "title": "GPT-4 Vision (Free Trial)",
      "provider": "free-trial",
      "model": "gpt-4-vision-preview"
    },
    {
      "title": "Gemini Pro (Free Trial)",
      "provider": "free-trial",
      "model": "gemini-pro"
    },
    {
      "title": "Codellama 70b (Free Trial)",
      "provider": "free-trial",
      "model": "codellama-70b"
    }, 
    {
      "title": "Code Llama",
      "provider": "ollama",
      "model": "codellama-7b"
    }
  ],
  "slashCommands": [
    {
      "name": "edit",
      "description": "Edit selected code"
    },
    {
      "name": "comment",
      "description": "Write comments for the selected code"
    },
    {
      "name": "share",
      "description": "Download and share this session"
    },
    {
      "name": "cmd",
      "description": "Generate a shell command"
    }
  ],
  "customCommands": [
    {
      "name": "test",
      "prompt": "Write a comprehensive set of unit tests for the selected code. It should setup, run tests that check for correctness including important edge cases, and teardown. Ensure that the tests are complete and sophisticated. Give the tests just as chat output, don't edit any file.",
      "description": "Write unit tests for highlighted code"
    }
  ],
  "contextProviders": [
    {
      "name": "diff",
      "params": {}
    },
    {
      "name": "open",
      "params": {}
    },
    {
      "name": "terminal",
      "params": {}
    },
    {
      "name": "problems",
      "params": {}
    },
    {
      "name": "codebase",
      "params": {}
    }
  ]
}
```

**Comment by sestinj** [comment]
@slice-sarthakd I've seen this in situations where the node version used to run VS Code is out-of-date. If you click the about menu, can you share what Node version it lists? You might have luck by installing the latest of VS Code, but if not let me know!


<img width="345" alt="Screenshot 2024-02-19 at 10 21 03 AM" src="https://github.com/continuedev/continue/assets/33237525/a3c304ab-0056-4d0b-a787-cc7f649413c5">

**Comment by slice-sarthakd** [comment]
Upgrading to latest vscode version solved it. Thanks!

**Comment by LoRAMilkshake** [comment]
I'm using the latest visual studio code and I'm having this problem

**Comment by Symbolk** [comment]
> @slice-sarthakd I've seen this in situations where the node version used to run VS Code is out-of-date. If you click the about menu, can you share what Node version it lists? You might have luck by installing the latest of VS Code, but if not let me know!
> 
> <img alt="Screenshot 2024-02-19 at 10 21 03 AM" width="345" src="https://private-user-images.githubusercontent.com/33237525/306046151-a3c304ab-0056-4d0b-a787-cc7f649413c5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1NDA4NTcsIm5iZiI6MTcyODU0MDU1NywicGF0aCI6Ii8zMzIzNzUyNS8zMDYwNDYxNTEtYTNjMzA0YWItMDA1Ni00ZDBiLWE3ODctY2M3ZjY0OTQxM2M1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDA2MDkxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI5MmM2YjNmNjQxNmMwNjA2MzBmMTkxMGE3NjgyYjM1MTYzNzMwMTdhMzI4MDllZTk0ZWU4NWE2ODc4OWNiZTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.M-pXn_YKwYWcwwU-imimS2FT9OQZgqk76GDgSZ-BCqc">

Yes, I have to use a customized VSCode, which is based on 1.81.3, with Node.JS 16.17.1. Any suggestion to solve this?
